### ISSUE TITLE
Is there a way to add custom SVG icons?

### ISSUE THREAD

**Issue by usefksa** [FortAwesome/react-fontawesome#243]
Dears,
Is there a way to add custom SVG icons?

**Comment by vdh** [comment]
Yes, as long as you create a definition file in the same format as the existing icon definitions, you can import and use them in the same way as the official icons. You may want to use a custom `prefix` for your own icons to keep them organised, though.

**Comment by tkrotoff** [comment]
Related issues:
- https://github.com/FortAwesome/Font-Awesome/issues/13079
- https://github.com/FortAwesome/Font-Awesome/issues/14742
